### DISCUSSION TITLE
Cannot login enable secret

### DISCUSSION THREAD

**Discussion by gusyee** [ktbyers/netmiko#2178] [Q&A]
How to login enable secret password device cisco with user priviliges 7?

**Comment by ktbyers** [comment]
Can you state more generally what you are trying to do? I.E. what is the broader automation objective you are trying to accomplish?

**Reply by gusyee** [reply]
I try to show run for backup config devices.

**Reply by ktbyers** [reply]
And are you executing `enable` and relying on AAA or are you executing `enable 7`?
